lib/bolt/outputter/human.rb

```diff
@@ -225,20 +226,26 @@ def print_task_info(task)
         # Building lots of strings...
         pretty_params = +""
         task_info = +""
-        usage = +"bolt task run --targets <node-name> #{task.name}"
+        usage = if Bolt::Util.powershell?
+                  +"Invoke-BoltTask -Name #{task.name} -Targets <targets>"
+                else
+                  +"bolt task run #{task.name} --targets <targets>"
+                end
 
         task.parameters&.each do |k, v|
           pretty_params << "- #{k}: #{v['type'] || 'Any'}\n"
           pretty_params << "    Default: #{v['default'].inspect}\n" if v.key?('default')
           pretty_params << "    #{v['description']}\n" if v['description']
-          usage << if v['type'].is_a?(Puppet::Pops::Types::POptionalType)
+          usage << if v['type'].start_with?("Optional")
```

Comment:
Why this change?

Reply:
Because it's a bug. `v['type']` is the stringified type, so `is_a?(Puppet::Pops::Types::POptionalType)` will always be false.